panel/lab/components/definitions/2_definition/index.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+	<k-lab-examples>
+		<k-lab-example label="Props">
+			<k-definitions>
+				<k-definition term="Term" description="Description" />
+				<k-definition term="Term" description="Description" />
+				<k-definition term="Term" description="Description" />
+			</k-definitions>
+		</k-lab-example>
+		<k-lab-example label="Slots">
+			<k-definitions>
+				<k-definition>
+					<template #term><em>Term</em></template>
+					<p class="k-text">
+						<a href="https://getkirby.com">Description</a>
+					</p>
+				</k-definition>
```

Comment:
Maybe also add a mixed example where the term gets passed as prop and the description via the default slot?